main.py

```diff
@@ -29,6 +29,8 @@
 from engine import evaluate, train_one_epoch, viz
 from models import build_model
 from models.backbone import build_swav_backbone, build_swav_backbone_old
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
We can add the same commands in the Deformable DETR repo as well, or otherwise remove it from here. Both are fine